### PR TITLE
My first pull request

### PR DESCRIPTION
**Pull request by ElsaGrace**
This is great, thanks for teaching me such a valuable life skill!

### CODE REVIEW

**Comment by ElsaGrace** [comment]
I don't understand what's happening with "all checks have failed". I just realized that I did not change the class name and updated it.

**Comment by ElsaGrace** [comment]
I just added my updated my distribution

File: distributions/Dist_yx1796.py [hunk 1]
```diff
@@ -4,20 +4,20 @@
...
-		self.x_min = -1
-		self.x_max = 1
+		self.x_min = 2
+		self.x_max = 4
 
 
 	def pdf(self, x):
 		"""This is your PDF"""
-		return np.abs(x)
+		return (x**3)/60
```
Comment (cranmer) [inline distributions/Dist_yx1796.py:13]: looks good

File: distributions/Dist_yx1796.py [hunk 2]
```diff
@@ -4,20 +4,20 @@
 class Dist_yx1796(BaseDistribution):
 	def __init__(self):
 		self.f_max = 1
```
Comment (cranmer) [inline distributions/Dist_yx1796.py]: I think the maximum of your function is x_max^4/60 = 64/60

**Comment by ElsaGrace** [comment]
Right I just changed the maximum to 64/60

**Pull request merged**